test/unittests/execute_call_test.cpp

```diff
@@ -58,6 +58,34 @@ TEST(execute_call, call_with_arguments)
     EXPECT_THAT(execute(parse(wasm), 1, {}), Result(4));
 }
 
+TEST(execute_call, call_shared_stack_space)
+{
+    /* wat2wasm
+    (module
+      (func $double (param i32) (result i64)
```

Comment:
It's hard to guess what these tests check... Is `i64` result required here? If not, maybe make it `i32` for simplicity and more focused test?